### ISSUE TITLE
show XCA federation vs centralization

### ISSUE THREAD

**Issue by JohnMoehrke** [IHE/HIE-Whitepaper#19]
with pro and con of each
This might be an independent whitepaper?

**Comment by JohnMoehrke** [comment]
this should be careful to be written for executive level audience, do not duplicate technical details found elsewhere and in the profile text. Should likely lean on the basic concepts of the definition of centralization and federation. 
If this text gets too long (more than a half sheet of paper), then it is a clear indication that we need a whole whitepaper on the topic.

**Comment by JohnMoehrke** [comment]
Centralized has an Initiating-Gateway/Responding-Gateway at the center. Where all traffic must traverse through this central actor. It has the benefit of having full knowledge of the traffic patterns, needs, usage, etc. Audit Logging can be maintained centrally. It also has the ability to 'fixup' requests/responses/content to satisfy dissonance in capability/conformance of various partners. For example: when one partner needs highly constrained content, where a responding partner is an older generation that does not meet those highly constrained content. Similar fixup can be done with security tokens. The drawback of this model is that provenance of all data and requests is reliant on the central system, the central system has full visibility to all protected health information, the central system can modify any request/response, the central system is a bottle-neck from a performance, and robustness.

**Comment by JohnMoehrke** [comment]
Federation enables each community to communicate within a trust domain. That trust domain is all that is managed centrally, which usually is a certificate authority(s) and organizational endpoint directory. All communications are between the partners, and are thus not exposed to any central system, thus no bottle-neck or provenance concern.
There is a perception that a federation is harder to maintain, but the central trust domain and configuration management simplifies that to the same degree as centralized.

**Comment by JohnMoehrke** [comment]
Should we cover a hybird where the basic federation is augmented with a Record Location Service (Patient Discovery) that is implemented centrally? That is to say that XCPD is always used with a central service, but XCA is federated?

**Comment by JohnMoehrke** [comment]
does more need to be said? 
Seems diagrams would help. I am not sure how to show in a diagram the difference. Need creative proposed diagrams

**Comment by JohnMoehrke** [comment]
@jlamy did you look at the comments on this issue and come to a conclusion of what should be said in the HIE whitepaper vs more detail that exists elsewhere?
The discussion lately on identity, centralized vs federated, did get me thinking that centralized vs federation does need to be highlighted. Where centralized everyone is trusting completely that central authority, federation is transaction and purpose specific trust. Meaning I can decide to be selective in my trust of a federated partner, but I must fully trust all the centralized partners. I think this is true of identity (XCPD) but also XCA like metadata

**Comment by JohnMoehrke** [comment]
@jlamy do you have a recommendation for text changes? Or do we consider the whitepaper is sufficient detail today?

**Comment by jlamy** [comment]
I like what is said now, but let's consider the following.

Section 3.3 merges the concepts of community and gateway, implying a federation consists of a bunch of communities that each have gateways. There are two counterexample use cases; should we mention them here?
- An XCA RG fronts multiple communities and exposes these in XCA query responses and potentially as parent-child relationships in a directory. The IG can either look for the community in the directory and retrieve directly from it, or retrieve from the original gateway. See ITI TF 2b: 3.38.4.1.3.2.
- An XCDR RG fronts multiple communities and exposes these as parent-child relationships in a directory. The IG, if it wants to direct a submission to a specific community, includes that HCID as routing information when submitting to the RG.

I notice that we never mention the concept of the Home Community ID, which I assume must be intentional. Nor do we ever connect the concepts of directories and communities, e.g. to say you could look up a community in a directory. Without this background, my cases above will be hard to follow for the reader.

If we wanted to allude to these cases without getting too technical, we could do the following:
- Modify section 3.3: "Behind each gateway may be a single organization, like the USA Veterans Administration, a complex community of many organizations, or a more simple organization like a single small hospital or facility. The gateway may hide the internal structure of each participant in the cross-community exchange, or may expose them as identifiable communities reachable through the gateway."
- Add a short section 3.4 Federated Push, patterned after 3.3:
3.4.1 Cross-Community Document Reliable Interchange (XCDR)
The XCDR profile provides a point-to-point method of sending documents to a specific community. It leverages common principles as described in Section 2. It sends documents and metadata using the same Web-Services reliable transport that is used to publish documents in the Centralized Discovery and Retrieve Document model. An XCDR Responding Gateway for a community may support submissions to multiple communities that are behind it. These communities would typically be described as parent-child relationships in a common directory.

**Comment by JohnMoehrke** [comment]
I like your high-level recommendation. The more detailed one that needs to use homeCommunityID is too deep.

I had not yet considered how XCDR could be mentioned. I guess your recommendation might be sufficient to bring that in too.

Please make these changes
